src/main/java/org/folio/service/file/storage/LocalFileSystemStorage.java

```diff
@@ -128,6 +128,9 @@ private String getFilePath(FileDefinition fileDefinition) {
 
   @Override
   public boolean isFileExist(String path) {
+    if (path == null) {
+      return false;
+    }
     return fileSystem.existsBlocking(path);
```

Comment:
Possible to return checking at same line path !=null and fileSystem.existsBlocking(path)

Reply:
Agree, thank you.